OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -298,7 +301,24 @@ public async Task UpdateWorkshop_WhenModelIsValid_ShouldReturnOkObjectResult()
             Assert.That(result, Is.Not.Null);
             Assert.AreEqual(Ok, result.StatusCode);
         }
+        #endregion
+
+        #region UpdateWorkshopStatus
+        [Test]
+        public async Task UpdateWorkshopStatus_WhenModelIsValid_ShouldReturnOkObjectResult()
```

Comment:
Word `Workshop` is already present  in the class name. Test name could be simplified.
```suggestion
        public async Task UpdateStatus_WhenModelIsValid_ShouldReturnOkObjectResult()
```